packages/form-js-viewer/src/render/components/Sanitizer.js

```diff
@@ -0,0 +1,155 @@
+const NODE_TYPE_TEXT = 3,
+      NODE_TYPE_ELEMENT = 1;
+
+const ALLOWED_NODES = [
+  'h1',
+  'h2',
+  'h3',
+  'h4',
+  'h5',
+  'span',
+  'em',
+  'a',
+  'p',
+  'div',
+  'ul',
+  'ol',
+  'li',
+  'hr',
+  'blockquote',
+  'img',
+  'pre',
+  'code',
+  'br',
+  'strong'
+];
+
+const ALLOWED_ATTRIBUTES = [
+  'align',
+  'alt',
+  'class',
+  'href',
+  'id',
+  'name',
+  'src',
+  'valign'
```

Comment:
What is this for since we don't allow `<table>` and friends?

Reply:
Oh. Sorry, I just added a comment about tables. 😅

Reply:
Removed via https://github.com/bpmn-io/form-js/commit/2fe2b202ca2c59588495931cefc3c886af907545.